src/libraries/LiquidatorLib.sol

```diff
@@ -32,34 +31,107 @@ library LiquidatorLib {
     uint256 amountOut0,
     uint128 reserve1,
     uint128 reserve0
-  ) internal pure returns (uint256 amountIn1) {
-    // require(amountOut0 > 0, "LiquidatorLib/insufficient-amount-out");
-    require(amountOut0 < reserve0, "LiquidatorLib/insufficient-reserve-liquidity");
-    require(reserve0 > 0 && reserve1 > 0, "LiquidatorLib/insufficient-reserve-liquidity");
+  ) internal view returns (uint256 amountIn1) {
+    require(amountOut0 < reserve0, "LiquidatorLib/insufficient-reserve-liquidity-c");
+    require(reserve0 > 0 && reserve1 > 0, "LiquidatorLib/insufficient-reserve-liquidity-d");
     uint256 numerator = amountOut0 * reserve1;
     uint256 denominator = uint256(reserve0) - amountOut0;
-    amountIn1 = (numerator / denominator);
+    amountIn1 = (numerator / denominator) + 1;
```

Comment:
Why we offset by 1 here?

Reply:
<https://github.com/Uniswap/v2-periphery/blob/0335e8f7e1bd1e8d8329fd300aea2ef2f36dd19f/contracts/libraries/UniswapV2Library.sol#L58>
To handle rounding when computing the amount in relative to an amount out. It ensures that truncation impact always goes in favour of the Liquidator - the user contributes more than necessary to get the requested `amountOut` rather than contributing less than necessary.